docs/docs/91-migrations.md

```diff
@@ -137,4 +157,5 @@ Migration from Drone is only possible if you were running Drone <= v0.8.
 
 1. Make sure you are already running Drone v0.8
 2. Upgrade to Woodpecker v0.14.4, migration will be done during startup
-3. Upgrade to the latest Woodpecker version. Pay attention to the breaking changes listed above.
+3. Upgrade to the latest Woodpecker version.
+   Pay attention to the breaking changes listed above.
```

Comment:
I think it's actually quite useful for reading and reviews.

Reading: Knowing exactly when a sentence ends by just looking at the text.
Reviews: Being able to address a specific sentence instead in a review (context-wise) and not having to say "I am actually referring to XY".

What are downsides?

Reply:
> Reading: Knowing exactly when a sentence ends by just looking at the text.

That's actually a main point why its so weird, it makes reading the text pretty difficult IMO as you always have to switch lines after just a few words. Kinda staccato sentences.

> Reviews: Being able to address a specific sentence instead in a review (context-wise) and not having to say "I am actually referring to XY".

I don't get this one. Works quite well to suggest only parts of a line regardless if its a sentence or multiple.

Reply:
IMO it's harder to read and the editor of choice needs automatic line breaks enabled, otherwise it becomes even harder to read. I also vote for disabling this rule.

Reply:
> That's actually a main point why its so weird, it makes reading the text pretty difficult IMO as you always have to switch lines after just a few words. Kinda staccato sentences.

OK, that might be a subjective point than with different views 😄 

> I don't get this one. Works quite well to suggest only parts of a line regardless if its a sentence or multiple.

Not the "suggestions" feature of GH (that is fine) but when you address the contextual meaning. IMO this only works well if it's *one* sentence and becomes way harder once a line consists of more sentences. 
Also, Gitea/CB does not have the selected suggestions feature so doing a suggestions in a PR there always requires one to c/p the exact sentence as otherwise it's unclear what the actual reference of review comment is. 

--- 

But OK, I guess people have different strong opinions here and we should probably not enforce one rule on it then. But you're aware that I've changed all "next" docs to the sentence-per-line rule in [4cad2cd](https://github.com/woodpecker-ci/woodpecker/pull/2669/commits/4cad2cdecccb7ac51c1fdbede0a9c4c7d1cedec8)? Should I cherry-pick remove this one again?

Reply:
Yes would prefer to undo those changes actually.

Reply:
I reverted related changes for now. In case we want those back we should be easily able to run the linter with the adjusted config again, right?